No such Property Field named: m_aARadar

While watching AI's fight:
```
Exception: No such Property Field named: m_aARadar, or: aARadar, for Subject: TechAttachment attached to:PlayerID named:Americans with name:techAttachment
java.lang.IllegalStateException: No such Property Field named: m_aARadar, or: aARadar, for Subject: TechAttachment attached to:PlayerID named:Americans with name:techAttachment
	at games.strategy.util.PropertyUtil.getPropertyFieldObject(PropertyUtil.java:76)
	at games.strategy.engine.data.ChangeAttachmentChange.<init>(ChangeAttachmentChange.java:29)
	at games.strategy.engine.data.ChangeFactory.attachmentPropertyChange(ChangeFactory.java:152)
	at games.strategy.triplea.delegate.TechTracker.addAdvance(TechTracker.java:81)
	at games.strategy.triplea.delegate.TechActivationDelegate.start(TechActivationDelegate.java:54)
	at games.strategy.engine.framework.ServerGame.startStep(ServerGame.java:510)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:418)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:274)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:63)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$171(AbstractLauncher.java:42)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalStateException: No such Property Field: aARadar
	at games.strategy.util.PropertyUtil.getFieldIncludingFromSuperClasses(PropertyUtil.java:57)
	at games.strategy.util.PropertyUtil.getFieldIncludingFromSuperClasses(PropertyUtil.java:63)
	at games.strategy.util.PropertyUtil.getFieldIncludingFromSuperClasses(PropertyUtil.java:63)
	at games.strategy.util.PropertyUtil.getFieldIncludingFromSuperClasses(PropertyUtil.java:63)
	at games.strategy.util.PropertyUtil.getFieldIncludingFromSuperClasses(PropertyUtil.java:53)
	at games.strategy.util.PropertyUtil.getPropertyField(PropertyUtil.java:85)
	at games.strategy.util.PropertyUtil.getPropertyFieldObject(PropertyUtil.java:70)
	... 10 more
```

--- a/src/games/strategy/triplea/attachments/TechAttachment.java
+++ b/src/games/strategy/triplea/attachments/TechAttachment.java
@@ -56,7 +56,7 @@ public class TechAttachment extends DefaultAttachment {
   private boolean increasedFactoryProduction = false;
   private boolean warBonds = false;
   private boolean mechanizedInfantry = false;
-  private boolean aaRadar = false;
+  private boolean aARadar = false;
   private boolean shipyards = false;
   // do not export at this point. currently map xml cannot
   // define a player having a custom tech at start of game
@@ -269,16 +269,16 @@ public class TechAttachment extends DefaultAttachment {
 
   @GameProperty(xmlProperty = true, gameProperty = true, adds = false)
   public void setAARadar(final String s) {
-    aaRadar = getBool(s);
+    aARadar = getBool(s);
   }
 
   @GameProperty(xmlProperty = true, gameProperty = true, adds = false)
   public void setAARadar(final Boolean s) {
-    aaRadar = s;
+    aARadar = s;
   }
 
   public void resetAARadar() {
-    aaRadar = false;
+    aARadar = false;
   }
 
   @GameProperty(xmlProperty = true, gameProperty = true, adds = false)
@@ -349,7 +349,7 @@ public class TechAttachment extends DefaultAttachment {
   }
 
   public boolean getAARadar() {
-    return aaRadar;
+    return aARadar;
   }
 
   public boolean getShipyards() {

